binary_search_tree/binary_search_tree.py

```diff
@@ -18,8 +18,30 @@ def insert(self, value):
           self.right.insert(value)
     else:
       self.value = value
+
   def contains(self, target):
-    pass
+    if target == self.value:
+      return True
+    elif target > self.value:
+      if self.right is None:
+        return False
+      elif target == self.right.value:
+        return True
+      else:
+        self.right.contains(target)
+    elif target < self.value:
+      if self.left is None:
+        return False
+      elif target == self.left.value:
+        return True
+      else:
+        self.left.contains(target)
+    else:
+      return False
+
 
   def get_max(self):
```

Comment:
This will return the tail of the list, not the max. You have to go through each node and store the maximum value you encounter as you go through the list.